### PR TITLE
Some minor listing cleanup from book 1 progression

### PR DESCRIPTION
**Pull request by hollasch**

### CODE REVIEW

**Review by armansito — APPROVED** [review]

File: books/RayTracingTheNextWeek.html [hunk 1]
```diff
@@ -71,9 +71,6 @@
 
 
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++ highlight
-        ray(const point3& origin, const vec3& direction) : orig(origin), dir(direction), tm(0)
```
Comment (armansito) [inline books/RayTracingTheNextWeek.html:74]: [src/TheNextWeek/ray.h](https://github.com/RayTracing/raytracing.github.io/blob/dev/src/TheNextWeek/ray.h) still defines both constructor overloads and `time` doesn't have a default value there. Does the source code need to get updated as well?

Comment (armansito) [reply books/RayTracingTheNextWeek.html:74]: Ah nvm, I see that the progression branch for book 2 has the change: https://github.com/RayTracing/raytracing.github.io/blob/progression/2/2023-08/src/TheNextWeek/ray.h

Comment (hollasch) [reply books/RayTracingTheNextWeek.html:74]: Looking at the end state source of book 2, I think this is another problem.

Comment (hollasch) [reply books/RayTracingTheNextWeek.html:74]: Yup. It is.

**Pull request merged**